app/controllers/hosts_controller.rb

```diff
@@ -151,7 +151,7 @@ def hostgroup_or_environment_selected
 
   def current_parameters
     Taxonomy.as_taxonomy @organization, @location do
-      render :partial => "common_parameters/inherited_parameters", :locals => {:inherited_parameters => refresh_host.host_inherited_params(true)}
+      render :partial => "common_parameters/inherited_parameters", :locals => {:inherited_parameters => refresh_host.inherited_params(true)}
```

Comment:
not sure I understand, I just moved host_inherited_params to host_common and changed it's name to inherited_params.